bazel/repositories.bzl

```diff
@@ -112,7 +112,7 @@ def envoy_api_deps(skip_targets):
     native.git_repository(
         name = "envoy_api",
         remote = REPO_LOCATIONS["envoy_api"],
-        commit = "9be6aff6da46e024af56cce20cb5d5d3184f19c5",
+        commit = "4add0b29e66adfce5df33754c922ac8cc496fb71",
```

Comment:
This will point at the hash for https://github.com/envoyproxy/data-plane-api/pull/178 after it is merged.